notanorm/__init__.py

```diff
@@ -8,5 +8,6 @@
 from .sqlite import SqliteDb
 from .base import DbBase
 from .model import DbType, DbCol, DbTable, DbModel, DbIndex
+from . import errors
```

Comment:
do we mean to put these in `__all__`?

Reply:
sticking it here just makes `import notanorm; notanorm.errors.IntegrityError` work.  I could stick them in __all__ too